tests/all-up-test.sh

```diff
@@ -25,4 +25,4 @@ ALCHEMY_ID=$2
 set -u
```

Comment:
Also, it would be nice to add a new script "[build-test-artifacts.sh]" which rebuilds the local artifacts.